perun-openapi/openapi.yml

```diff
@@ -8900,3 +9075,265 @@ paths:
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/tasksManager/getTaskResultsForGUIByTask:
+    get:
+      tags:
+        - TasksManager
+      operationId: getTaskResultsForGUIByTask
+      summary:  Return list of TaskResults by a Task for GUI.
+      parameters:
+        - $ref: '#/components/parameters/taskId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfTaskResultsResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/tasksManager/getTaskResultById:
+    get:
+      tags:
+        - TasksManager
+      operationId: getTaskResultById
+      summary:  Returns TaskResult by its id.
+      parameters:
+        - $ref: '#/components/parameters/taskResultId'
+      responses:
+        '200':
+          $ref: '#/components/responses/TaskResultResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/tasksManager/getTaskById:
+    get:
+      tags:
+        - TasksManager
+      operationId: getTaskById
+      summary:  Returns Task by its id
+      parameters:
+        - $ref: '#/components/parameters/id'
+      responses:
+        '200':
+          $ref: '#/components/responses/TaskResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/tasksManager/getFacilityState:
+    get:
+      tags:
+        - TasksManager
+      operationId: getFacilityState
+      summary:   Return propagation status of facility.
+      parameters:
+        - $ref: '#/components/parameters/facilityId'
+      responses:
+        '200':
+          $ref: '#/components/responses/FacilityStateResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/tasksManager/getAllFacilitiesStates:
+    get:
+      tags:
+        - TasksManager
+      operationId: getAllFacilitiesStates
+      summary: Return propagation status of all facilities in Perun/Vo.
+      parameters:
+        - $ref: '#/components/parameters/optionalVoId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfFacilityStatesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/tasksManager/getAllResourcesState:
+    get:
+      tags:
+        - TasksManager
+      operationId: getAllResourcesState
+      summary: Return propagation status of all resources related to VO.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfResourceStatesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/tasksManager/getTaskResultsForDestinations:
+    get:
+      tags:
+        - TasksManager
+      operationId: getTaskResultsForDestinations
+      summary: Returns task results for defined destinations.
+      parameters:
+        - name: destinations
```

Comment:
If you send a list of values in the URL parameters (and not in the request body), the name must end with the `[]` brackets.